lib/unsupported-features/node-globals.js

```diff
@@ -135,14 +135,15 @@ const nodeGlobals = {
     // module.crypto
     crypto: {
         ...crypto.webcrypto,
-        [READ]: { experimental: ["17.6.0", "16.15.0"] },
+        [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] },
     },
-    Crypto: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
-    CryptoKey: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
+    Crypto: { [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] } },
+    CryptoKey: { [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] } },
     SubtleCrypto: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
 
     // module.events
-    CustomEvent: events.CustomEvent,
+    CloseEvent: { [READ]: { supported: ["23.0.0"] } },
+    CustomEvent: {[READ]: { experimental: ["18.7.0", "16.17.0"], supported: ["23.0.0"] } },
```

Comment:
Global `CustomEvent` differs from `node:event`'s `CustomEvent`

- https://nodejs.org/docs/latest-v23.x/api/globals.html#customevent
  ![](https://github.com/user-attachments/assets/91c2d377-1522-4d05-8026-90c824d8b475)
- https://nodejs.org/docs/latest-v23.x/api/events.html#class-customevent
  ![](https://github.com/user-attachments/assets/6fb5fba1-ab69-45bc-9018-2404ec17cfd9)

Reply:
:exploding_head: I mostly look at the global docs page, that could be annoying...

I wonder if we should ask someone from Node themselves?

Reply:
Indeed annoying, but that's what the docs are saying. 🤷‍♂️

I think we should always stick to what the docs are saying.

I do think it would be a good idea to have a discussion with people on the Node team to find out if docs are incorrect and if so, update the docs (and our code).

Reply:
I see thanks for the explanation here :bow:: https://github.com/eslint-community/eslint-plugin-n/pull/358#discussion_r1806583886